FinSearchUnified/Subscriber/Frontend.php

```diff
@@ -150,20 +172,19 @@ public function onFrontendPostDispatch(Enlight_Event_EventArgs $args)
         $searchResultContainer = Shopware()->Config()->get('SearchResultContainer');
         $navigationContainer = Shopware()->Config()->get('NavigationContainer');
 
-        try {
```

Comment:
This try/catch should stay too.

Reply:
There was no Enlight_Exception being thrown in the code. Should I change it to a Exception instead?

Reply:
Yes, please. In any error case this shouldn't break the shop but fail silently.